### DISCUSSION TITLE
Quest Request: One way street, is it one way for bicycle too?

### DISCUSSION THREAD

**Discussion by patepelo** [streetcomplete/StreetComplete#4957] [Ideas]
In my area (Barcelona, Spain) it is very common to find these signs for one way streets except bicycle 

![image](https://user-images.githubusercontent.com/23003127/233125977-6afa944d-13c6-4e3a-936d-1d0ff43d76a3.png)

So checking manually on Vespucci to see if the "Key:oneway:bicycle=no" is added and then adding it takes lots of effort. Can we add this quest for: highway=living_street and maybe highway=residential?

Checking it should either update to: oneway:bicycle=yes or no. So we know this feature has been updated already.

**Comment by Helium314** [comment]
You can check it using the cycleway overlay, and thus use SC instead of Vespucci.
But currently there is no color indicating the oneway state.

**Comment by matkoniecz** [comment]
Was discussed in https://github.com/streetcomplete/StreetComplete/issues/3778 and had some problems but making overlay more useful for this case is also possible.

Have you tried using cycleway overlay already for this use case?

**Reply by patepelo** [reply]
![Screenshot_20230419-193742.png](https://user-images.githubusercontent.com/23003127/233155823-d1186e0c-b2fa-4d90-abc5-16a6aa5dde99.png)

How do i set that up? I have only Bike paths? See image

**Reply by matkoniecz** [reply]
Yes, it shows also bicycle lanes and contraflow. Maybe better name would be a good idea.
